Lib/test/test_codecs.py

```diff
@@ -1713,6 +1714,14 @@ def test_undefined(self):
             self.assertRaises(UnicodeError,
                 codecs.decode, b'abc', 'undefined', errors)
 
+    def test_file_closes_if_lookup_error_raised(self):
+        mock_open = mock.mock_open()
+        with mock.patch('builtins.open', mock_open) as file:
+            with self.assertRaises(LookupError):
+                codecs.open(support.TESTFN, mode = 'wt', encoding='invalid-format')
```

Comment:
Too long line and spaces around "=".

Actually using keywords for passing argument is not needed.

```suggestion
                codecs.open(support.TESTFN, 'wt', 'invalid-encoding')
```

Reply:
Thanks! Committed the suggested change above.